hi-ml-azure/src/health_azure/utils.py

```diff
@@ -104,7 +104,6 @@
     "MKL_SERVICE_FORCE_INTEL": "1",
     # Switching to a new software stack in AML for mounting datasets
     "RSLEX_DIRECT_VOLUME_MOUNT": "true",
-    "RSLEX_DIRECT_VOLUME_MOUNT_MAX_CACHE_SIZE": "1",
```

Comment:
deprecated according to AML team

Reply:
We can probably also retire the other RSLEX... thing

Reply:
Oh no actually, RSLEX_DIRECT_VOLUME_MOUNT might be essential. I quote from azure doc 
"For read only mount environment variable RSLEX_DIRECT_VOLUME_MOUNT="true" should be set before starting mount."
"For writable mount environment variable RSLEX_DIRECT_VOLUME_WRITABLE_MOUNT="true" should be set before starting mount."

Not sure, what the default is, but to be on the safe side let's keep it read only.